components/hero.tsx

```diff
@@ -33,34 +36,63 @@ export default function Hero() {
         {/* Hero content */}
         <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
           {/* Section header */}
-          <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Pear: The Open-Source AI-Powered Code Editor
-            </h1>
-            <p
-              className="mb-8 text-xl text-secondary-main"
-              data-aos="fade-up"
-              data-aos-delay="200"
-            >
-              Supercharge your development in an editor designed for less
-              coding, with AI.
-            </p>
+          <div className="mx-auto flex max-w-3xl flex-col items-center pb-12 text-center md:pb-16">
+            <div className="flex flex-col space-x-2 text-3xl font-semibold md:items-center md:text-4xl">
+              <div className="flex flex-row items-center space-x-2">
+                <PearHeroLogo width="26" className="hidden sm:block" />
+                <PearHeroLogo width="22" className="sm:hidden" />
+                <span className="text-primary-700">PearAI:</span>
+                <span className="sm:hidden">The Open Source </span>
+                <span className="hidden sm:block">
+                  The Open Source AI-Powered
+                </span>
+              </div>
+              <span className="sm:hidden">AI-Powered Code Editor</span>
+              <span className="hidden sm:block">Code Editor</span>
+              {/*
+            </h1> */}
+            </div>
+            <div className="mt-8 max-w-md sm:max-w-lg">
+              <p
+                className="mb-2 text-sm text-secondary-500 sm:text-lg"
+                data-aos="fade-up"
+                data-aos-delay="200"
+              >
+                Speed up your development process by seamlessly integrating AI
+                into your workflow 🚀
+              </p>
+              <p
+                className="mb-4 text-sm text-secondary-500 sm:text-lg"
+                data-aos="fade-up"
+                data-aos-delay="200"
+              >
+                Afraid of switching editors? No need, Pear is a fork of VSCode
+                and retains all its features, you'll feel right at home 😏
```

Comment:
replace `you'll` with `you&apos;ll` otherwise Eslint will throw error

Reply:
Done!